testproject/tests/test_second_step_authentication.py

```diff
@@ -3,31 +3,65 @@
 from django.contrib.auth import get_user_model
 from django.contrib.auth.hashers import make_password
 
+import time
 from rest_framework.test import APIClient
 from twilio.base.exceptions import TwilioException, TwilioRestException
 
 from tests.utils import (
+    PATH_AUTH_JWT_LOGIN,
+    PATH_AUTH_JWT_LOGIN_CODE,
     get_token_from_response,
     get_username_from_jwt,
     header_template,
     login,
 )
-from trench.utils import create_otp_code, generate_backup_codes
+from trench.backends.provider import get_mfa_handler
+from trench.command.generate_backup_codes import generate_backup_codes_command
 
 
 User = get_user_model()
 
 
+@pytest.mark.django_db
+def test_custom_validity_period(active_user_with_email_otp, settings):
+    settings.TRENCH_AUTH["MFA_METHODS"]["email"]["VALIDITY_PERIOD"] = 3
+
+    client = APIClient()
+    first_step = login(active_user_with_email_otp)
+    handler = get_mfa_handler(mfa_method=active_user_with_email_otp.mfa_methods.first())
+    code = handler.create_code()
+    time.sleep(3)
+    response = client.post(
+        path=PATH_AUTH_JWT_LOGIN_CODE,
+        data={
+            "ephemeral_token": first_step.data.get("ephemeral_token"),
+            "code": code,
+        },
+        format="json",
+    )
+    assert response.status_code == 401
```

Comment:
in other tests we are using `rest_framework.status` const so we can do in this test too